src/drive/mobile/styles/uploadprogression.styl

```diff
@@ -1,12 +1,22 @@
-@require 'cozy-ui'
+.coz-upload-status {
+    height 48px
+    padding 5px 16px
+    box-sizing border-box
+    border-bottom 1px solid #d6d8da
+    transition bottom 0.5s ease-out, top 0.5s ease-out, opacity 0.2s ease-out
+    display flex
+}
```

Comment:
There are still curly braces which we don't have elsewhere :(

Reply:
I can't understand why there is such difference between css syntax and stylus syntax. What is the purpose? Where is the automatic format tool?

Reply:
I'm not even sure it's stylus per se, since stylus totally understands regular css syntax as well. I personally dislike it too — it doesn't change anything in terms or readability, and it actually takes me more time to write it like that.
You're right though, there should be an automatic formatting tool for this.

Reply:
The purpose is to write less syntax, more code. But I agree I'v never been fond of this syntax myself. Besides, my editor kinda do the CSS syntax automatically for me…